src/TetherContent.js

```diff
@@ -26,7 +28,10 @@ class TetherContent extends React.Component {
     super(props);
 
     this.handleDocumentClick = this.handleDocumentClick.bind(this);
+    this.checkTogglerTarget = this.checkTogglerTarget.bind(this);
```

Comment:
This does not appear to be needed since `this.checkTogglerTarget` only appears to be called from `this.handleDocumentClick` which will always have the correct context of `this`